.fixtures.yml

```diff
@@ -0,0 +1,7 @@
+fixtures:
+  repositories:
+    stdlib:
+      repo: https://github.com/puppetlabs/puppetlabs-stdlib.git
+      ref: 4.25.0
+  symlinks:
```

Comment:
adding the last two lines isn't required anymore. puppetlabs_spec_helper will do this for us

Reply:
Ah good to know, I removed those lines.